### PR TITLE
GEOMESA-1917 Install Jopt-simple with install-kafka scripts

### PR DESCRIPTION
**Pull request by jnh5y**
Signed-off-by: Jim Hughes <jnh5y@ccri.com>

### CODE REVIEW

File: geomesa-lambda/geomesa-lambda-tools/bin/install-kafka.sh [hunk 1]
```diff
@@ -16,6 +16,7 @@ declare -a urls=(
   "https://search.maven.org/remotecontent?filepath=org/apache/kafka/kafka-clients/$kafka_version/kafka-clients-$kafka_version.jar"
   "https://search.maven.org/remotecontent?filepath=org/apache/zookeeper/zookeeper/$zookeeper_version/zookeeper-$zookeeper_version.jar"
   "https://search.maven.org/remotecontent?filepath=com/101tec/zkclient/$zkclient_version/zkclient-$zkclient_version.jar"
+  "https://search.maven.org/remotecontent?filepath=net/sf/jopt-simple/jopt-simple/%%jopt-simple.version%%/jopt-simple-%%jopt-simple.version%%.jar"
```
Comment (elahrvivaz) [inline geomesa-lambda/geomesa-lambda-tools/bin/install-kafka.sh]: probably don't need this for lambda if it's `listen`-specific

Comment (jnh5y) [inline geomesa-lambda/geomesa-lambda-tools/bin/install-kafka.sh]: Have we tried all the commands with geomesa-lambda?

Comment (elahrvivaz) [inline geomesa-lambda/geomesa-lambda-tools/bin/install-kafka.sh]: i think so... i did some testing with the script and some with KAFKA_HOME so can't verify 100%

Comment (elahrvivaz) [inline geomesa-lambda/geomesa-lambda-tools/bin/install-kafka.sh]: seems like the dep comes from the listen command invoking some kafka command line classes, i don't think lambda does anything like that

**Comment by jnh5y** [comment]
Ok, I've nuked the jopt bits for the lambda datastore.

**Pull request merged**